src/screens/splash.rs

```diff
@@ -35,12 +36,23 @@ pub(super) fn plugin(app: &mut App) {
         )
             .run_if(in_state(Screen::Splash)),
     );
+
+    // Exit the splash screen early if the player hits escape.
+    app.add_systems(
+        Update,
+        exit_splash_screen
+            .run_if(input_just_pressed(KeyCode::Escape).and_then(in_state(Screen::Splash))),
```

Comment:
I personally think that this reads better when written as two separate `run_if` because of the line break, but I'm not blocking on that.

Reply:
IMO that's confusing because you have to know whether that yields an `and` or an `or` behavior.